src/sql/base/browser/ui/table/tableCellEditorFactory.ts

```diff
@@ -42,6 +47,7 @@ export class TableCellEditorFactory {
 			private _originalValue: string;
 			private _input: InputBox;
 			private _keyCaptureList: number[];
+			private _toDispose = new DisposableStore();
```

Comment:
Any reason not to just make the TextEditor disposable itself and then dispose it when it's destroyed?